test/tests/bcs/ad_1d_neumann/1d_neumann.i

```diff
@@ -0,0 +1,46 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 10
+  construct_side_list_from_node_list = true
+[]
+
+[Variables]
+  [./u]
+  [../]
+[]
+
+[Kernels]
+  [./diff]
+    type = Diffusion
+    variable = u
+  [../]
+[]
+
+[BCs]
+  [./left]
+    type = DirichletBC
+    variable = u
+    boundary = left
+    value = 0
+  [../]
+  [./right]
+    type = ADNeumannBC
+    variable = u
+    boundary = right
+    value = 2
+  [../]
+[]
+
+[Executioner]
+  type = Steady
+
+  solve_type = 'PJFNK'
```

Comment:
Don't we want these tests to be `NEWTON`?

Reply:
yup!